integrationtest/org/daisy/dotify/formatter/test/resource-files/marker-ref/marker-reference-spread-start-workaround-input.obfl

```diff
@@ -2,43 +2,41 @@
 <obfl xmlns="http://www.daisy.org/ns/2011/obfl" version="2011-1" xml:lang="und" hyphenate="false">
    <meta>
       <dc:title xmlns:dc="http://purl.org/dc/elements/1.1/">marker-reference-spread-start-workaround</dc:title>
-      <dc:description xmlns:dc="http://purl.org/dc/elements/1.1/">Tests (hypothetical) marker-reference type that corresponds with 'spread-start' in CSS. This type is not directly supported in OBFL. As a workaround I do a forward spread search for the marker 'foo/prev' starting on the left page (rendered top left corner), and if that does not return a value, a backward sequence search for the marker 'foo' starting on the right page (top right corner). At any position in the OBFL the value of the preceding marker of class 'foo' matches the value of the following marker of class 'foo/prev'. In addition, tests marker-reference with scope spread and direction backward, starting on the right page, which corresponds more or less with 'spread-last-except-start' in CSS (bottom left corner). Everything tested on both left- and right-hand pages. Tested with markers at the beginning of the sequence, in the middle of a page, at a soft page break within a block, at a soft page break between two blocks, and at a hard page break.</dc:description>
+      <dc:description xmlns:dc="http://purl.org/dc/elements/1.1/">Tests (hypothetical) marker-reference type that corresponds with 'spread-start' in CSS (top left corner). This type is not directly supported in OBFL. As a workaround I do a forward spread search for the marker 'foo/prev' starting on the left page, and if that does not return a value, a backward sequence search for the marker 'foo' starting on the right page. At any position in the OBFL the value of the preceding marker of class 'foo' matches the value of the following marker of class 'foo/prev'. In addition, tests marker-reference with scope spread and direction backward, starting on the right page, which corresponds more or less with 'spread-last-except-start' in CSS (bottom left corner). Everything tested on both left- and right-hand pages. Tested with markers at the beginning of the sequence, in the middle of a page, at a soft page break within a block, at a soft page break between two blocks, and at a hard page break.</dc:description>
```

Comment:
This file has "workaround" in its name. Is that still actual?

Reply:
Changed the name.